src/test/java/com/homihq/db2rest/utils/ITestUtil.java

```diff
@@ -0,0 +1,222 @@
+package com.homihq.db2rest.utils;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class ITestUtil {
```

Comment:
This is static data (almost module like).  I'd likely rename this to something like `/request-FilmData.java`

Reply:
and instead of a path like `src/test/java/com/homihq/db2rest/utils/ITestUtil.java` perhaps be named and moved to a concise path of `src/test/data/RequestFilmData.java`

Reply:
@thadguidry  please log an issue. This can be a good first issue :) 
I am not sure at this time but JUNIT 5 probably allows loading test data from external sources better than earlier versions. 
In that case, test data like this can be sourced from JSON files instead.

Reply:
And if we are rubbing against the old Maven folder conventions, then we likely don't have our POM setup correctly so that we can have good concise paths.

Reply:
@thadguidry @kdhrubo  The integrations tests were a mess. Extracting test data into an util class was the first step, next we'll take further steps to organise the test data. I thought about having separate test json files and load those files as resources during tests, which'll be the next step